index.ts

```diff
@@ -1,2 +1,46 @@
-import CashBalances from './data';
+import CashBalances, {CashBalanceReport, ReportSection, Account} from './data';
+
+interface ReportAccountTotals {
+  highest_account_name: string
+  lowest_account: string
+  average: any
+}
+
+/**
+ * Takes a report section and returns a flattened list of accounts
+ */
+function getFlattenedListOfAccounts(section: ReportSection): Account[] {
+  return [...section.accounts,
+    ...section.nestedSection[0].accounts, ...section.nestedSection[0].nestedSection[0].accounts,
+    ...section.nestedSection[0].nestedSection[0].nestedSection[0].accounts]
+}
```

Comment:
This could be ok, but I'm missing some context here. Will this data ever change and will we only ever have to flatten four levels?